app/views/provider_mailer/_international_relocation_payment.text.erb

```diff
@@ -0,0 +1,3 @@
+Based on the subject this candidate is applying to train to teach, they may be entitled to £10,000 to help with the financial costs of moving to England. ‘Get an international relocation payment’ will be disbursed with the support of teacher training providers on the bursaries model.
+
+To learn more about candidate eligibility, how it will be checked and what the IRP means for providers, please visit [LINK] or email us at [teach.inengland@education.gov.uk](mailto:teach.inengland@education.gov.uk).
```

Comment:
Is this `[LINK]` placeholder?

Reply:
yeah waiting on confirmation from @MaeveRoseveare